tests/chunk_test.py

```diff
@@ -28,56 +32,58 @@ def _chunk(response, split):
 
 
 def test_get_text():
-    pool = mocket.MocketPool()
-    pool.getaddrinfo.return_value = ((None, None, None, None, (ip, 80)),)
-    c = _chunk(text, 33)
-    print(c)
-    sock = mocket.Mocket(headers + c)
-    pool.socket.return_value = sock
+    for extra in (b"", b";blahblah; blah"):
```

Comment:
Mind copying and pasting this into a new test function? That will lead to a clearer failure if/when it fails. Right now it's lumped into the basic case.

Reply:
I've updated the test, let me know if it's still not what you're looking for.